src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const CAR_COST = 40;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const PRICE = days * CAR_COST;
+
+  if (days >= LONG_TERM) {
+    return PRICE - LONG_TERM_DISCOUNT;
```

Comment:
The logic for applying the long-term discount is correct, as it checks if the rental days are greater than or equal to the LONG_TERM threshold. However, ensure that the discount is applied correctly for rentals of 7 days or more.